docs/getting_started.md

```diff
@@ -5,6 +5,9 @@
 
 1. Python 3.7
 2. AWS Account credentials when testing with AWS platforms
+   check default section in ~/.aws/credentials for access/secret key.
```

Comment:
Maybe you want to link to https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html